debian/patches/drop-unsupported-systemd-condition-environment.patch

```diff
@@ -25,27 +25,17 @@ This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
  ConditionPathExists=!/etc/cloud/cloud-init.disabled
  ConditionKernelCommandLine=!cloud-init=disabled
 -ConditionEnvironment=!KERNEL_CMDLINE=cloud-init=disabled
- 
- 
+
+
  [Service]
 --- a/systemd/cloud-init-local.service.tmpl
 +++ b/systemd/cloud-init-local.service.tmpl
-@@ -29,7 +29,6 @@ Conflicts=shutdown.target
+@@ -28,7 +28,6 @@ Conflicts=shutdown.target
  RequiresMountsFor=/var/lib/cloud
  ConditionPathExists=!/etc/cloud/cloud-init.disabled
  ConditionKernelCommandLine=!cloud-init=disabled
 -ConditionEnvironment=!KERNEL_CMDLINE=cloud-init=disabled
- 
- [Service]
- Type=oneshot
---- a/systemd/cloud-init.service.tmpl
-+++ b/systemd/cloud-init.service.tmpl
```

Comment:
@TheRealFalcon This actually drops the patch from the re-named cloud-init.service.tmpl (now named cloud-init-network.service.tmpl). I can add it back in in my followup PR if you want - otherwise we should rename it and I'll rename it back. I'm fine with doing that either way - let me know.

Reply:
Yeah, we're kind if in a weird "post single process PR" but "pre patch it out" state. I think I'm fine leaving it as-is as you'll need to make changes in your patch either way.

Reply:
Are we good to merge this PR then?

Reply:
Go for it!